src/lib/project.js

```diff
@@ -332,7 +339,42 @@ async function scaffoldNext() {
   }
 };`
   );
+  newNextConfig = newNextConfig.replace(
+    'reactStrictMode: true',
+    'reactStrictMode: false'
+  );
   fs.writeFileSync(path.join('ui', 'next.config.js'), newNextConfig);
+
```

Comment:
FYI here is the default tsconfig generate when we create a next typescript project.

```
CompilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
  "exclude": ["node_modules"]
}
```